.github/workflows/ci_macos_mu4.yml

```diff
@@ -223,10 +223,6 @@ jobs:
       if: env.DO_PUBLISH == 'true'
       run: |
         bash ./buildscripts/ci/tools/osuosl/publish.sh -s ${{ secrets.OSUOSL_SSH_ENCRYPT_SECRET }} --os macos -v 4
-    - name: AppCast
-      if: env.DO_BUILD == 'true'
-      run: |
-        bash ./buildscripts/ci/tools/sparkle_appcast_gen.sh -p macos   
```

Comment:
@igorkorsukov Or do we still need this to notify MS3 users about new versions of MS4?

Reply:
Hm, if so, that isn't working in MuseScore 3.6.2 at least, because that shows a simple dialog mentioning MS4.0.0 with a link that downloads Muse Hub. 
Perhaps it still works in MS3.5.x, because at that point Sparkle wasn't broken yet.